.github/workflows/goreleaser.yml

```diff
@@ -75,9 +75,10 @@ jobs:
 
       - name: Run GoReleaser
         id: run-goreleaser
-        uses: goreleaser/goreleaser-action@8f67e590f2d095516493f017008adc464e63adb1 # v4.1.0
+        uses: goreleaser/goreleaser-action@v6 # v4.1.0
```

Comment:
Can we please pin the version to the SHA? This is a safeguard against malicious actions because the tag `v6` may be updated to point to a malicious version of the action. Pinning to SHA sum ensures immutability of the action.